### ISSUE TITLE
list_devices pairing template also shows already paired devices in fw 0.10.1

### ISSUE THREAD

**Issue by gruijter** [athombv/homey#938]
**What is your Homey version (Settings → System)?**

0.10.1

**What did you try to achieve?**

Opening the device pairing template and show a list of devices that can be paired

*Please include detailed steps how we can reproduce the issue*

in the driver this is used:
```socket.on('list_devices', function( data, callback ){  ```

and in the callback all available devices are sent using:
``` callback( null, deviceList ) ```

**What was the result?**

The complete devicelist is shown in the frontend, and all devices can be selected to pair with, even when the device has already been paired with
...

**What did you expect as result?**

That only devices that haven't been paired with are shown.
...

**Any other remarks**
Example of the devicelist I am producing:
```
listing of devices started
entered makeDeviceList
[ { name: 'Vaatwasser',
    data: { id: '000D6F000037A850' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Wasmachine',
    data: { id: '000D6F000038D1D3' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Haardroger',
    data: { id: '000D6F000038BE51' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'PV schuin dak',
    data: { id: '000D6F0000D364F9' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'TV Hoek',
    data: { id: '000D6F0001A40677' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Autolader',
    data: { id: '000D6F0000D3412B' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Koelkast',
    data: { id: '000D6F000038D2D9' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Wasdroger',
    data: { id: '000D6F00003847C9' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Koffiezetter',
    data: { id: '000D6F000037A836' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'PV plat dak',
    data: { id: '000D6F00003846B5' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Werkkamer',
    data: { id: '000D6F0001A5A14B' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] },
  { name: 'Thijs kamer',
    data: { id: '000D6F0001A44608' },
    capabilities: [ 'onoff', 'measure_power', 'meter_power' ] } ]
```
...

**Comment by WeeJeWel** [comment]
Thanks for reporting! Fixed in 0.10.2

**Comment by jeroenvollenbrock** [comment]
I'm closing this issue as it is fixed in the latest update. If the problem persists, please reopen this issue.
